fv3config/_datastore.py

```diff
@@ -40,6 +42,10 @@ def set_cache_dir(parent_dirname):
 
 
 def get_cache_dir():
+    return USER_CACHE_DIR
+
+
+def _get_cache_subdir():
     return os.path.join(USER_CACHE_DIR, CACHE_PREFIX)
```

Comment:
Could you describe the motivation behind this change? It is not clear to me what is a `_cache_subdir` from the name alone.

Reply:
Maybe `_internal_cache_dir` would work. Or just copy-paste your last comment into that  the docstring of `_cache_subdir`.

Reply:
My hope was that "_" would make the "internal" part obvious. Nonetheless, I've renamed the function to `_get_internal_cache_dir`. It's a bit of a long name now which detracts from readability a bit, but the explicitness probably cancels that out.

Reply:
ok. thanks.